modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Project/index.tsx

```diff
@@ -104,6 +104,7 @@ const ProjectPermissions: React.FC<ProjectsProps> = (props) => {
 		'CREATE',
 		'UPDATE',
 		'DELETE',
+		'VIEW'
```

Comment:
You do not need to commit this change since the 'INDEX' constant do the same thing.